qhub/cost.py

```diff
@@ -0,0 +1,128 @@
+    """
+    try:
+        process = subprocess.Popen(
+            ["infracost", "breakdown", "--path", path, "--format", "json"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+        stdout, stderr = process.communicate()
+        infracost_data = json.loads(
+            re.search("({.+})", stdout.decode("UTF-8"))
+            .group(0)
+            .replace("u'", '"')
+            .replace("'", '"')
+        )
+        return infracost_data
+    except subprocess.CalledProcessError:
+        return None
+    except AttributeError:
+        return None
+
+
+def infracost_report(path):
+    """
+    Generate a report of the infracost cost of the given path
+    args:
+        path: path to the qhub stages directory
+    """
+    if not path:
+        path = os.path.join(os.getcwd(), "stages")
+
+    if _check_infracost() and _check_infracost_api_key():
+        if not os.path.exists(path):
+            print("Deployment is not available")
+        else:
+            data = _run_infracost(path)
+            if data:
+                cost_table = Table(title="Cost Breakdown")
+                cost_table.add_column(
+                    "Name", justify="right", style="cyan", no_wrap=True
+                )
+                cost_table.add_column(
+                    "Cost ($)", justify="right", style="cyan", no_wrap=True
+                )
+
+                cost_table.add_row("Total Monthly Cost", data["totalMonthlyCost"])
+                cost_table.add_row("Total Hourly Cost", data["totalHourlyCost"])
+
+                resource_table = Table(title="Resource Breakdown")
+                resource_table.add_column(
+                    "Name", justify="right", style="cyan", no_wrap=True
+                )
+                resource_table.add_column(
+                    "Number", justify="right", style="cyan", no_wrap=True
+                )
+
+                resource_table.add_row(
+                    "Total Detected Costs",
+                    str(data["summary"]["totalDetectedResources"]),
+                )
+                resource_table.add_row(
+                    "Total Supported Resources",
+                    str(data["summary"]["totalSupportedResources"]),
+                )
+                resource_table.add_row(
+                    "Total Un-Supported Resources",
+                    str(data["summary"]["totalUnsupportedResources"]),
+                )
+                resource_table.add_row(
+                    "Total Non-Priced Resources",
+                    str(data["summary"]["totalNoPriceResources"]),
+                )
+                resource_table.add_row(
+                    "Total Usage-Priced Resources",
+                    str(data["summary"]["totalUsageBasedResources"]),
+                )
+
+                console = Console()
+                console.print(cost_table)
+                console.print(resource_table)
```

Comment:
I think it would be neat to add the ability to write to csv and/or png, but that's just an idea. I don't want to hold up this PR for that.

Reply:
Perhaps `console.print` like you have above?